pkg/controller/pxc/users.go

```diff
@@ -151,7 +220,7 @@ func (r *ReconcilePerconaXtraDBCluster) manageSysUsers(cr *api.PerconaXtraDBClus
 		pxcPass = string(internalSysSecretObj.Data["operator"])
 	}
 
-	um, err := users.NewManager(cr.Name+"-pxc."+cr.Namespace, pxcUser, pxcPass)
+	um, err := users.NewManager(cr.Name+"-pxc-unready."+cr.Namespace+":33062", pxcUser, pxcPass)
```

Comment:
why we don't pass a port as a separate argument?

Reply:
because driver configuration take only full address https://github.com/percona/percona-xtradb-cluster-operator/blob/proxy-protocol-update/pkg/pxc/users/users.go#L28